src/detail-page/components/charts/DevOpsChart.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { Pie } from 'react-chartjs-2';
+import { devOps } from '../../../data/charts';
+import { chartColors } from '../../../data/charts/chartColors';
+import { ChartWrapper } from './ChartWrapper';
+import { notEmpty } from '../../../shared/utils/helpers';
+
+const { data: sourceData, labelText, source, title: chartTitle, type } = devOps;
+
+const chartLabels = sourceData.map(({ label }) => label);
+const chartValues = sourceData.map(({ value }) => value);
+const chartBackgroundColors = sourceData
+  .map(({ color }) => (color ? chartColors[color].light : null))
```

Comment:
We could get rid of one pair of parenthesis here :blush: `.map(({ color }) => color ? chartColors[color].light : null)` would be enough

Reply:
I've corrected it. My prettier sometimes has strange ideas ;)